ui/package.json

```diff
@@ -29,7 +29,8 @@
     "precommit": "pretty-quick --staged",
     "start": "PORT=4400 react-scripts start",
     "build": "react-scripts build",
-    "test": "jest tests/e2e",
+    "test-1000-genomes": "jest tests/e2e-1000-genomes",
```

Comment:
Let's have `npm test` test 1000 genomes. If someone wants to manually run tests, 1000 Genomes will be good enough most of the time.
```
"test": "jest tests/e2e-1000-genomes",
```